doc/nrf/ug_thingy91_gsg.rst

```diff
@@ -15,6 +15,7 @@ If you have already set up your Thingy:91 and want to learn more, see the follow
 
 * :ref:`ug_thingy91` for more advanced topics related to the Thingy:91.
 * The :ref:`introductory documentation <getting_started>` for more information on the |NCS| and the development environment.
+* :ref:`connectivity_bridge` about using the nRF UART Service with Thingy:91.
```

Comment:
I wouldn't add an entry here. It's already pointing to Developing with, which does mention it with this update. 

Maybe in the bottom Next steps section?